packages/nc-gui/utils/iconUtils.ts

```diff
@@ -31,9 +83,61 @@ export const iconMap = {
   'mdi-thumb-up-outline': MdiThumbUpOutline,
   'mdi-flag': MdiFlag,
   'mdi-flag-outline': MdiFlagOutline,
-  'mdi-table-large': MdiTableLarge,
-  'mdi-eye-circle-outline': MdiEyeCircleOutline,
-  'mdi-account-group': MdiAccountGroup,
+  'mdi-table-large': PhTableThin,
+  'mdi-eye-circle-outline': PhEyeThin,
+  'mdi-account-group': PhUsersThreeThin,
+
+  'lock-icon': PhLockIcon,
```

Comment:
for the naming, would it be better to use `lockIcon` instead of `lock-icon`? so that we can use `iconMap.lockIcon` instead of `iconMap['lock-icon']` which looks more consistent?

Reply:
make sense, I will update them